types/storybook-readme/index.d.ts

```diff
@@ -2,10 +2,12 @@
 // Project: https://github.com/tuchk4/storybook-readme
 // Definitions by: Taeheon Kim  <https://github.com/lonyele>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.1
+
+import { ComponentType, ReactNode } from 'react';
 
 // Shared Types
-export type Renderable = React.ComponentType | JSX.Element;
+export type Renderable = ComponentType | JSX.Element;
```

Comment:
For `storybook-readme` related issues (https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/604709995#L588) I'm sure we can ask @lonyele some help to update this typings to make them work with SB own types ;)